Packages/com.unity.inputsystem/InputSystem/Plugins/XR/Devices/OpenVR.cs

```diff
@@ -51,7 +51,7 @@ protected override void FinishSetup()
         }
     }
 
-    [InputControlLayout(commonUsages = new[] { "LeftHand", "RightHand" })]
+    [InputControlLayout(displayName = "OpenVR Controller (Windows MR)", commonUsages = new[] { "LeftHand", "RightHand" })]
```

Comment:
This one should follow the same pattern as Oculus' 'Oculus Touch Controller (OpenVR)'
Maybe
Windows Spatial Controller (OpenVR)
?

Reply:
Fixed. Dropped the "Spatial" for brevity. Hope that's okay.